ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml

```diff
@@ -19,6 +23,8 @@ images:
       as:
       - registry.svc.ci.openshift.org/openshift/release:golang-1.12
   to: elasticsearch-proxy
+- from: elasticsearch-operator-src
+  to: elasticsearch-operator-e2e
```

Comment:
not sure why you're doing this instead of just referencing elasticsearch-operator-src below in the test definition?

is that tag not showing up as available?